src/game/Entities/Player/Player.cpp

```diff
@@ -23162,12 +23162,12 @@ void Player::ApplyEquipCooldown(Item* pItem)
             continue;
 
         // xinef: apply hidden cooldown for procs
-        if (spellData.SpellTrigger == ITEM_SPELLTRIGGER_ON_EQUIP)
+        /*if (spellData.SpellTrigger == ITEM_SPELLTRIGGER_ON_EQUIP)
         {
             // xinef: uint32(-1) special marker for proc cooldowns
             AddSpellCooldown(spellData.SpellId, uint32(-1), 30*IN_MILLISECONDS);
             continue;
-        }
+        }*/
```

Comment:
if this code is really unneeded, then it should be actually **removed** and not just commented

Reply:
Not sure why xinef would add that it's best to comment it out.